apps/docs/content/guides/platform/manage-your-usage/storage-image-transformations.mdx

```diff
@@ -0,0 +1,83 @@
+---
+id: 'manage-usage-storage-image-transformations'
+title: 'Manage Storage Image Transformations usage'
+---
+
+## What you are charged for
+You are charged for the number of images transformed during the billing period, regardless of how many transformations each image undergoes. We refer to these images as **origin** images.
+
+**Example:** If you apply four different size transformations to one image and a single transformation to another, the total count is two, as only two origin images were transformed.
+
+## How charges are calculated
+Storage Image Transformations are billed using Package pricing, with each package representing 1000 origin images. If your usage falls between two packages, you are billed for the next whole package.
```

Comment:
This could contain some examples of actual storage requests and how transforming the same image with different parameters only counts as 1

Reply:
Extended the example